src/components/ha-cover-tilt-controls.html

```diff
@@ -0,0 +1,110 @@
+<link rel='import' href='../../bower_components/polymer/polymer.html'>
+
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+<link rel='import' href='../../bower_components/paper-slider/paper-slider.html'>
+
+<link rel='import' href='../util/cover-model.html'>
+
+<dom-module id='ha-cover-tilt-controls'>
+  <template>
+    <style is="custom-style" include="iron-flex"></style>
+    <style>
+      :host {
+        --ha-cover-tilt-controls-display: inline-block;
+        display: var(--ha-cover-tilt-controls-display);
+      }
+      
+      .tilt {
+        white-space: nowrap;
+        max-height: 0px;
+      }
+
+      .has-open_tilt .tilt,
+      .has-close_tilt .tilt,
+      .has-stop_tilt .tilt,
+      {
+        max-height: 40px;
```

Comment:
The `max-height` classes here are redundant with the invisible attribute. We use `max-height` on the more info dialog so that hidden controls collapse vertically, but here the buttons should just occupy their space anyway, so visibility is enough.